Inventory.cs

```diff
@@ -8,31 +8,169 @@ namespace JanasInventoryManagementSystem
 {
     public class Inventory
     {
-        private List<Product>? products = new List<Product>();
+        private static List<Product>? products = new List<Product>();
 
-        public void Add(Inventory inventory)
+        public void Add()
         {
+            Console.Write("Enter product name: ");
+            string name = Console.ReadLine();
 
+            Console.Write("Enter product price: ");
+            decimal price = Convert.ToDecimal(Console.ReadLine());
+
+            Console.Write("Enter product quantity: ");
+            int quantity = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine();
+
+            Product product = new Product(name, price, quantity);
+            products.Add(product);
```

Comment:
When editing a Product, we rely on the product name, meaning it must be unique in the collection. Before allowing or denying an addition, check if the name already exists. Consider the performance implications of your implementation approach. Research the most efficient collection type to use for this purpose, focusing on how to perform quick lookups and avoid duplicates.